src/vivarium_gates_mncnh/model_specifications/model_spec.yaml

```
@@ -157,6 +158,9 @@ configuration:
                 - 'pregnancy_outcome'
                 - 'azithromycin_availability'
                 - 'misoprostol_availability'
+    propensity_ordering:
+        antenatal_care: ['none', 'later_pregnancy_only', 'first_trimester_only', 'first_trimester_and_later_pregnancy']
```

Is this more readable?
```suggestion
        antenatal_care: 
            - 'none',
            - 'later_pregnancy_only',
            - 'first_trimester_only',
            - 'first_trimester_and_later_pregnancy'
```